README.md

```diff
@@ -37,63 +50,66 @@ Start the server using:
 
-* `app/controllers/kittens_controller.rb`
-* `app/views/kittens/index.html.erb`
+* `app/controllers/kittens_controller.rb` - main controller with `:index` entry point
+* `app/views/kittens/index.html.erb` - main template for kittens index view
+
+The controllers work as expected for Rails applications.
 
-Controllers work as usual in Rails applications.
 
 ## Assets
 
-Relevant files:
+The relevant files for the assets are:
 
-* `lib/open_project/engine.rb` - assets statement at the end of the engine
-* `lib/open_project/hooks.rb` - The Javascript and Stylesheet are included through here.
-* `app/javascripts/proto_plugin/main.js`
-* `app/stylesheets/proto_plugin/main.scss`
-* `app/assets/images/kitty.png`
+* `lib/open_project/proto_plugin/engine.rb` - assets statement at the end of the engine.
+* `lib/open_project/proto_plugin/hooks.rb` - the JavaScript and Stylesheet are included here.
+* `app/assets/javascripts/proto_plugin/main.js` - main entry point for plain JavaScript and document ready hook.
+* `app/assets/stylesheets/proto_plugin/main.scss` - good ol' Sass stuff.
+* `app/assets/images/kitty.png` - a nice kitty image.
 
-Any assets you want to use have to be registered for pre-compilation in the engine like this:
+Any additional assets you want to use have to be registered for pre-compilation in the engine like this:
 
 ```
 assets %w(proto_plugin/main.css proto_plugin/main.js kitty.png)
 ```
 
-You don't technically have to put the assets into a subfolder named after your plugin.
-But it's recommended to avoid naming conflicts. The image (`kitty.png`) is not scoped and would conflict with the core if it were to include an asset called `kitty.png` too.
+You don't technically have to put the assets into a subfolder with the same name as your plugin. But it's highly recommended to do so in order to avoid naming conflicts. For example, if the image `kitty.png` is not scoped, it might conflict with the core if it were also to include another asset named `kitty.png` too.
+
 
 ## Frontend
 
-Relevant files:
+The relevant files for the frontend are:
 
 * `package.json`
 * `frontend/app/openproject-proto_plugin-app.js`
 * `frontend/app/controllers/kittens.js`
 * `app/views/kittens/index.html.erb`
 
-If you want to work within the frontend's angular app you will need to provide a `package.json`.
-Take a look at the `frontend` folder to see an example Angular controller which is used in the "kittens" index view.
+If you want to work within the frontend's AngularJS app you will need to provide a `package.json`. Take a look at the `frontend` folder to see an example Angular controller which is used in the "kittens" index view.
 
-Changes in the frontend require running webpack to update. For that run the following in the OpenProject folder (not the plugin):
+Any changes made to the frontend require running webpack to update. For that run, go to the OpenProject folder (NOT the plugin directory) and execute the following command:
```

Comment:
The "run, " doesn't make sense here anymore like this, does it?
I suggest changing the beginning of the sentence to "To do that go to the OpenProject folder ...".